app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -998,7 +1132,7 @@ const renderSubjectWithDefaultValues = (onSubmit = () => {}) =>
         firefoxMinVersion: NimbusExperimentFirefoxVersionEnum.NO_VERSION,
         firefoxMaxVersion: NimbusExperimentFirefoxVersionEnum.NO_VERSION,
         channel: NimbusExperimentChannelEnum.NO_CHANNEL,
-        populationPercent: "0.0",
+        populationPercent: "0",
```

Comment:
I am curious to know why you made this change?? as the decimal are expected value 🤔

Reply:
The default value is now defined as "0" :)